enhancements/storage/storage-hypershift.md

```diff
@@ -0,0 +1,973 @@
+
+##### cluster-storage-operator inputs
+Apart from the objects mentioned in the previous chapter, the operator accesses following API objects:
+
+| Object            | Where | Description                                                            |
+|-------------------|-------|------------------------------------------------------------------------|
+| Nodes             | G     | The operator needs nr. of nodes to scale the operand correctly on SNO. |
+| Infrastructure CR | G     | The operator needs to know on which cloud it runs.                     |
+| FeatureGates CR   | G     | To enable optional CSI driver operators.                               |
+
+##### aws-ebs-csi-driver-operator inputs
+Apart from the objects mentioned in the previous chapters, the operator accesses following API objects:
+
+| Object                           | Where | Description                                                            |
+|----------------------------------|-------|------------------------------------------------------------------------|
+| Nodes                            | G     | The operator needs nr. of nodes to scale the operand correctly on SNO. |
+| Infrastructure CR                | G     | The operator needs to know on which cloud it runs and its parameters.  |
+| ConfigMap with cluster CA bundle | G     | To inject it into the CSI driver Pods.                                 |
+| ConfigMap with cloud CA bundle   | G     | To inject it into the CSI driver Pods.                                 |
+| ConfigMap with kube-cloud-config | G     | To get AWS config (e.g. the CA bundle above).                          |
+| Nodes                            | G     | To set parameters for SNO.                                             |
+| Proxy                            | G     | To inject cluster-wide proxy to the CSI driver.                        |
+
+##### Required changes for AWS EBS CSI driver in HyperShift
+
+* **control-plane-operator**:
+  * Create ServiceAccount (SA) for the operator.
+    * Add any RBAC if needed, the SA will create Deployment, PDBs, SAs, Secrets, ConfigMaps, ... in the same namespace
+      as control-plane-operator.
+  * Create cluster-storage-operator-kubeconfig with kubeconfig for the operator + operand + 2nd level operand to
+    the guest cluster.
+    * cluster-admin is OK in Phase1.
+    * Phase 2:
+      * Create RBACs for cluster-storage-operator and aws-ebs-csi-driver-operator in the guest cluster + generate
+        separate kubeconfigs for them. Sharing the same YAML files as in the standalone cluster would be very
+        beneficial, so we don't need to update the RBACs in two places.
+  * Create a role in AWS for the CSI driver and generate a Secret with STS credentials of the operator and the driver
```

Comment:
This already happens in hypershift

Reply:
See above - is it in the management cluster?

Reply:
As explained above, they don't need to necessarily be in the management cluster.